ansible/todoapp.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=My To-Do App
+
+[Service]
+User=ec2-user
+WorkingDirectory=/opt/todoapp/
+EnvironmentFile=/opt/todoapp/.env
+ExecStart=/home/ec2-users/.local/bin/poetry run gunicorn -b 0.0.0.0:5000 "todo_app.app:create_app()"
+
+[Install]
+WantedBy=multi-user.target
```

```suggestion
[Unit]
Description=My To-Do App

[Service]
User=todoapp
WorkingDirectory=/opt/todoapp/
EnvironmentFile=/opt/todoapp/.env
ExecStart=/home/todoapp/.local/bin/poetry run gunicorn -b 0.0.0.0:5000 "todo_app.app:create_app()"

[Install]
WantedBy=multi-user.target
```
I thought this wouldn't quite work, but I can see the actual deployed version is different! Suggestion above just matches the current running version